src/components/PokemonType.js

```diff
@@ -7,7 +7,7 @@ export default function PokemonType({ passedStyle, type }) {
   const typeColor = colors.pokemon.types[type];
   return (
     <View style={[styles.container, { backgroundColor: typeColor }, passedStyle]}>
-      <Text style={[styles.content, { color: colors.white }]}>{type.toUpperCase()}</Text>
+      <Text style={[styles.content, { color: colors.white }, passedStyle]}>{type.toUpperCase()}</Text>
     </View>
   );
```

Comment:
I think if we want to change text component's style we should also change props to differentiate styles
```
<View style={[styles.container, { backgroundColor: typeColor }, containerStyle]}>
    <Text style={[styles.content, { color: colors.white }, textStyle]}>{type.toUpperCase()}</Text>
</View>
```